hypertrace-view-generator/helm/templates/view-creation/all-views/all-views-creation.yaml

```diff
@@ -1,8 +1,8 @@
-{{- if .Values.mergeAllViews }}
+{{- if .Values.mergeAllViewsCreation }}
```

Comment:
Lets keep this as view creation job and lets remove all individual view creation job.

So, basically `.Values.mergeAllViewsCreation = true` and removes individual job like `backend-entity-view-creation`

Reply:
done